serialization/json/examples/agent1.json

```diff
@@ -0,0 +1,19 @@
+{
+  "@context": "https://spdx.github.io/spdx-3-model/rdf/context.json",
+  "@type": "Agent",
+  "@id": "https://some.namespace#agent1",
+  "creationInfo": {
+    "specVersion": "3.0.0",
+    "created": "2022-12-01T00:00:00",
+    "createdBy": ["https://spdx.dev/elements/3F26391C#spdx-dev"],
+    "profile": ["core"],
+    "dataLicense": "https://spdx.org/licenses/CC0-1.0"
+  },
+  "name": "John Smith",
+  "externalIdentifier": [
+    {
+      "externalIdentifierType": "email",
```

Comment:
Similarly, shouldn't this be `http://spdx.org/rdf/v3/Core/ExternalIdentifierType/email` ?

Reply:
[This line](https://github.com/spdx/tools-python/blob/7881f7ebe74261142555da1027e1cd6ca1049268/src/spdx_tools/spdx3/writer/json_ld/context.json#L230) and the following take care of that.